core/math/math_funcs.h

```diff
@@ -427,13 +432,23 @@ class Math {
 		float s = CLAMP((p_s - p_from) / (p_to - p_from), 0.0f, 1.0f);
 		return s * s * (3.0f - 2.0f * s);
 	}
+
 	static _ALWAYS_INLINE_ double move_toward(double p_from, double p_to, double p_delta) {
 		return abs(p_to - p_from) <= p_delta ? p_to : p_from + SIGN(p_to - p_from) * p_delta;
 	}
 	static _ALWAYS_INLINE_ float move_toward(float p_from, float p_to, float p_delta) {
 		return abs(p_to - p_from) <= p_delta ? p_to : p_from + SIGN(p_to - p_from) * p_delta;
 	}
 
+	static _ALWAYS_INLINE_ double move_toward_angle(double p_from, double p_to, double p_delta) {
+		double difference = angle_difference(p_from, p_to);
+		return wrapf(p_from + fmin(abs(difference), p_delta) * SIGN(difference), -Math_PI, Math_PI);
+	}
+	static _ALWAYS_INLINE_ float move_toward_angle(float p_from, float p_to, float p_delta) {
+		float difference = angle_difference(p_from, p_to);
+		return wrapf(p_from + fmin(abs(difference), p_delta) * SIGN(difference), -(float)Math_PI, (float)Math_PI);
```

Comment:
```suggestion
		return Math::wrapf(p_from + MIN(Math::abs(difference), p_delta) * SIGN(difference), -(float)Math_PI, (float)Math_PI);
```
We're explicit about the methods from this same class, also use the Godot specific minimum function not the global scope one

Same applies for the line above

Reply:
I'd also suggest prefixing our own functions here with `Math::` as that's partially adhered to

Reply:
Done, I've seen a little bit of inconsistency with the former but that's probably for another PR.